README.md

```diff
@@ -69,9 +72,9 @@ jobs:
     us-docker.pkg.dev/my-project/my-container/image:1.2.3
     ```
 
--   `source`: (Required, unless providing `metadata` or `image`) Path to source
-    to deploy. If specified, this will deploy the Cloud Run service from the
-    code specified at the given source directory.
+-   `source`: (Required, unless providing `metadata`, `image`, or `job`) Path to
+    source to deploy. If specified, this will deploy the Cloud Run service from
+    the code specified at the given source directory.
```

Comment:
```suggestion
-   `source`: (Required, unless providing `metadata`, `image`, or `job`) Path to
    source to deploy. If specified, this will deploy the Cloud Run service or job from
    the code specified at the given source directory.
```